### PR TITLE
Configure linters

### PR DESCRIPTION
**Pull request by vickymarz**
### Linters Configuration

_In this project;_

- I setup Html-CSS linters
- I practiced documenting my work in a professional way
- I added files to gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+
+## Acknowledgments
+
+- Microverse
+- Coding Partners
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Hamzaoutdoors) [inline README.md:57]: - [ ] The **MIT** link is broken. Kindly fix this. Your `README` is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users ^_^

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @vickymarz, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README similar to the Microverse design

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
...
+
+### Prerequisites
+- Have VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Install node package. [Link to download node](https://nodejs.org/en/download/)
+- Have git installed.[Link to download git](https://git-scm.com/downloads)
+- Create a github repository.
+
+### Setup
+- git clone httpd://github.com/your/your-project.git
+- cd your-project/
```
Comment (elmar8287) [inline README.md:27]: - [ ] [OPTIONAL] It will be better if you write here your project name. Actually you use the template - just modify by your repo

**Review by elmar8287 — APPROVED** [review]
Hi @vickymarz   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉🎉🎉 

**I have marked an OPTIONAL suggestion in a comment under review. Please, consider it for your future projects.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (vickymarz) [reply README.md:27]: ok

**Pull request merged**